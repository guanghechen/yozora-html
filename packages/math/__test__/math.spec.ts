import type { Math } from '@yozora/ast'
import renderMath from '../src'

describe('snapshot', function () {
  it('basic', function () {
    const math: Math = {
      type: 'math',
      value: '$$x^2 + y^2 = z^2$$',
    }
    expect(renderMath(math)).toMatchSnapshot()
  })
})
