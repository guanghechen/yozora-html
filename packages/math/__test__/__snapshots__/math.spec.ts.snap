// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<div class=\\"yozora-math\\">$$x^2 + y^2 = z^2$$</div>"`;
